Assets/MRTK/SDK/Features/UX/Prefabs/RiggedHandVisualizer/RiggedHandLeft.prefab

```diff
@@ -33,25 +32,6 @@ Transform:
   m_Father: {fileID: 0}
   m_RootOrder: 0
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
---- !u!95 &61451597030450500
-Animator:
```

Comment:
Why does the animator need to be removed?

Reply:
The animators weren't doing anything on the prefabs (and in fact had missing references on them)